.github/actions/repo-gardening/src/send-slack-message.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * External dependencies
+ */
+const fetch = require( 'node-fetch' );
+
+/* global WebhookPayloadPullRequest */
+
+/**
+ * Send a message to a Slack channel using the Slack API.
+ *
+ * @param {string}                    message - Message to post to Slack
+ * @param {string}                    channel - Slack channel ID.
+ * @param {string}                    token   - Slack token.
+ * @param {WebhookPayloadPullRequest} payload - Pull request event payload.
+ *
+ * @returns {Promise<boolean>} Promise resolving to a boolean, whether message was successfully posted or not.
+ */
+async function sendSlackMessage( message, channel, token, payload ) {
```

Comment:
I think we should name this something more specific like "sendSlackReviewRequest".

Reply:
I thought about keeping this more generic so we can use it for other types of Slack messages later on. I realize the CTA still says "Review" though. What if the CTA said "View" or "Click to see"? What do you think?

Reply:
In that case I'd still lean towards something like "sendSlackMessageForPR". But either way, I won't let this one hold up approving.